BUILD.md

```diff
@@ -5,13 +5,13 @@
 Install dependencies:
 
 ```sh
-sudo apt install -y podman dh-python python3 python3-stdeb python3-pyside2.qtcore python3-pyside2.qtgui python3-pyside2.qtwidgets python3-appdirs python3-click python3-xdg python3-colorama
+sudo apt install -y podman dh-python python3 python3-pip python3-stdeb python3-pyside2.qtcore python3-pyside2.qtgui python3-pyside2.qtwidgets python3-appdirs python3-click python3-xdg python3-colorama
```

Comment:
Nice catch! I wanted to update these instructions as well, based on some recent tests I did in stripped-down environments. Besides `python3-pip`, I see that these packages were missing as well:

* `python3-dev` (Debian-only)
* `python3-venv` (Debian-only)
* `make` (Debian and Fedora)

Did you by any chance have to install them? If not, are they maybe already present in your environment?

Reply:
In a Qubes Debian environment, `make` was already there. `python3-dev` is a dependency of `python3-pip`, so even through it was not present, it was installed anyways. But perhaps we can list that explicitly.

`python3-venv` doesn't appear to be needed (we only use it [in the CI](https://github.com/freedomofpress/dangerzone/blob/main/.circleci/config.yml#L82)).

> **Note:** after this PR we should take a look at the CI config to make sure they-re in since. We have too many package  dependencies spread around [multiple](https://github.com/freedomofpress/dangerzone/blob/main/.circleci/config.yml#L17) places [in](https://github.com/freedomofpress/dangerzone/blob/main/stdeb.cfg#L4) the [codebase](https://github.com/freedomofpress/dangerzone/blob/main/BUILD.md?plain=1#L8).

Reply:
Got it, thanks for the dig!